spec/features/sample_manifests/uploader_for_manifests_with_tag_sequences_spec.rb

```diff
@@ -97,7 +97,7 @@
           visit('sample_manifest_upload_with_tag_sequences/new')
           attach_file('File to upload', test_file)
           click_button('Upload manifest')
-          expect(page).to have_content('Your sample manifest couldn\'t be uploaded. See errors below.')
+          expect(page).to have_content('Your sample manifest couldn\'t be uploaded.')
```

Comment:
Was this just because the errors are now displayed differently, or do you no longer see a breakdown?

Reply:
The former. Because of the refactoring in sample_manifest_upload_with_tag_sequences_controller